### ISSUE TITLE
Temporary Access Token for a Robot Account

### ISSUE THREAD

**Issue by raviranjandutta** [goharbor/harbor#18595]
I'm trying to get a temporary access token of robot account for docker login through cli so that i can push or pull image.

my question is :
1> is it possible to get temp access token.
if yes, what is the curl to get that. if there is any example or document i can check at.

FYI, I'm running a harbor container in local.

thank you

**Comment by raviranjandutta** [comment]
harbor version is latest docker image

**Comment by raviranjandutta** [comment]
does someone have answer for this. please comment

**Comment by wy65701436** [comment]
no, harbor doesn't provide temp access token for robot account. I will make this as a feature request.

**Comment by nidhey27** [comment]
By **temporary access token**, You mean an access token that would have an **expiration time**, ensuring that the token is only valid for a certain period of time.
@raviranjandutta @wy65701436

**Comment by raviranjandutta** [comment]
yes have some expiration time say a min something

**Comment by raviranjandutta** [comment]
i am trying to docker login by robotacct and its password but it doesn't allow me to login. is that the expected behavior

**Comment by nidhey27** [comment]
I'm interested to work on this feature.

**Comment by wy65701436** [comment]
> I'm interested to work on this feature.

You can file the proposal firstly at goharbor/community, and then the maintainers will take effort to review it.

**Comment by nidhey27** [comment]
@wy65701436 Please allow me some time to work on this. It's been more than a year since I last looked into the code, so I need to get familiar with it again before filing the proposal.

**Comment by wy65701436** [comment]
> @wy65701436 Please allow me some time to work on this. It's been more than a year since I last looked into the code, so I need to get familiar with it again before filing the proposal.

sure, take your time and let me know if anything is needed.
